### PR TITLE
Update Chat Prompts and Increment Versions

### PR DESCRIPTION
**Pull request by gdyre**

### CODE REVIEW

File: assets/large_language_models/rag/components/validate_deployments/spec.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ type: command
 tags:
     Preview: ""
 
-version: 0.0.35
+version: 0.0.36
```
Comment (prakharg-msft) [inline assets/large_language_models/rag/components/validate_deployments/spec.yaml:7]: why this change?

**Review by gjwoods — APPROVED** [review]
